lib/instance.js

```diff
@@ -22,10 +22,9 @@ var initValues = function(values, options) {
     defaults = {};
 
     if (this.Model._hasDefaultValues) {
-      Utils._.each(this.Model._defaultValues, function(valueFn, key) {
-        if (defaults[key] === undefined) {
-          defaults[key] = valueFn();
-        }
+      defaults = _.mapValues(this.Model._defaultValues, function(valueFn) {
+        var value = valueFn();
+        return value._isSequelizeMethod ? value : _.cloneDeep(value);
```

Comment:
The `_isSequelizeMethod` check does not work for me. Even if it did, I still think we should "cloneDeep" regardless if a function was given or not. This also protects users from passing a function and returning the same object reference.

Suggested implementation:

```js
      defaults = _.mapValues(this.Model._defaultValues, function(fn) {
        return _.cloneDeep(fn());
      });
```